src/Fixer/Import/OrderedImportsFixer.php

```diff
@@ -474,11 +474,11 @@ private function getNewOrder(array $uses, Tokens $tokens)
     }
 
     /**
-     * @param $indexes
+     * @param array[] $indexes
```

Comment:
wow, this is array of arrays? really? what's inside? could you give example?

Reply:
They are built here: https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/760f404720db334e4a52bde5daf83123ea152f2e/src/Fixer/Import/OrderedImportsFixer.php#L410-L416

Reply:
the alternative to avoid this was `continue 2;` IIRC ;)

Reply:
no, solution is to use real DTO and never use assoc arrays